gauntlet/src/lib.rs

```diff
@@ -133,6 +133,19 @@ pub struct Args {
     pub shellcheck: bool,
 }
 
+/// Normalizes diagnostic message text to handle platform-specific differences.
+/// This ensures the same diagnostic message on different platforms is treated
+/// as identical.
+pub fn normalize_diagnostic(s: &str) -> String {
+    let s = s.replace('\\', "/").replace("\r\n", "\n");
+
+    // Handle OS-specific error messages
+    s.replace(
+        "The system cannot find the file specified. (os error 2)",
```

Comment:
The error in question is actually different from the one the file tests see.

On Windows, error code 2 is for "file not found" and error code 3 is for "path not found". Gauntlet is running into "path not found" errors because the import paths from `broad/warp` contain a directory that doesn't exist.

So I think we'll need this:

```suggestion
        "The system cannot find the path specified. (os error 3)",
```